### PR TITLE
Add string for Error notifications

### PR DESCRIPTION
**Pull request by rosepuppy**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dydxprotocol/v4-localization",
-  "version": "1.1.46",
+  "version": "1.1.47",
```
Comment (moo-onthelawn) [inline package.json:3]: hm should we add back the deleted blurb from [here](https://github.com/dydxprotocol/v4-localization/pull/326/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519L3)? or do you think it's probably ok 👀

File: dydxprotocol-v4-localization-1.1.46.tgz
Comment (moo-onthelawn) [inline dydxprotocol-v4-localization-1.1.46.tgz:1]: what's the tgz needed for?

File: package.json (hunk 1 above)
Comment (rosepuppy) [reply package.json:3]: looks like probably ok I think

File: dydxprotocol-v4-localization-1.1.46.tgz
Comment (rosepuppy) [reply dydxprotocol-v4-localization-1.1.46.tgz:1]: oh good catch, shouldn't be here

File: package.json (hunk 1 above)
Comment (rosepuppy) [reply package.json:3]: not totally sure what it is for either

**Review by ruixhuang — APPROVED** [review]

**Pull request merged**